src/main/java/com/rackspace/ceres/app/model/MetricNameAndMultiTags.java

```diff
@@ -0,0 +1,12 @@
+package com.rackspace.ceres.app.model;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.Data;
+
+@Data
+public class MetricNameAndMultiTags {
+    String metricName;
+    List<Map<String, String>> tags;
```

Comment:
We need to support multiple entries with same key. It is easier implementation and easier to understand.
e.g
host=h-1,
host=h-2,
host=*

Reply:
Is each map storing more than one value? Could we make it a list of Pair's? https://docs.oracle.com/javase/9/docs/api/javafx/util/Pair.html

Reply:
A bit reluctant to create a dependency on oracle. I think map works ok.

Reply:
Oh yeah didn't realize that when I linked it. Good call.